solr/packaging/test/test_ssl.bats

```diff
@@ -54,8 +54,8 @@ teardown() {
   solr start -c
   solr assert --started https://localhost:8983/solr --timeout 5000
 
-  run curl --cacert "$ssl_dir/solr-ssl.pem" 'https://localhost:8983/solr/admin/collections?action=CREATE&collection.configName=_default&name=test&numShards=2&replicationFactor=1&router.name=compositeId&wt=json'
-  assert_output --partial '"status":0'
+  run solr create -c test -s 2
```

Comment:
Did you mean this change?    I guess this is demonstrating that bin/solr create works with https?    When you define those variables?

Reply:
I looked, and I see another curl example elsewhere in the tests.  I actually like the example of using curl with the `solr-ssl.pem`, it helped me understand how to interact with Solr in SSL ;-)

Reply:
Yep, it's to test the usage!